tests/cases/whitespace/trailing

```diff
@@ -4,11 +4,11 @@ Must not have trailing whitespace
 <?php 
   
 declare(strict_types=1);   
-    
-function foo(array $bar) : iterable     
+
+function foo() : void
 {      
-    foreach ($bar as $baz) {       
-        yield $baz;        
```

Comment:
Looks like this test has been invalidated, the whitespace has been trimmed

Reply:
Heh. :shakes-fist-at-helpful-ide: